gnome-base/gnome-control-center/metadata.xml

```diff
@@ -8,6 +8,7 @@
   <use>
     <flag name="gnome-online-accounts">Enable configuration panel for <pkg>net-libs/gnome-online-accounts</pkg> accounts</flag>
     <flag name="ibus">Enable support for enhanced input methods through <pkg>app-i18n/ibus</pkg></flag>
+    <flag name="location">Build with location services</flag>
```

Comment:
We have the global "geolocation" USE flag that looks to also fit for this usage

Thanks

Reply:
Done in https://github.com/gentoo/gentoo/pull/34760/commits/f8d531d31f37e8dde7edd4f7233089148e12c094